frontend/src/app/sessions/new-readonly-session-dialog/new-readonly-session-dialog.component.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, Inject, OnInit } from '@angular/core';
+
+import { MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { Router } from '@angular/router';
+
+import { GetGitModel } from 'src/app/projects/project-detail/model-overview/model-detail/git-model.service';
+import { GitService } from 'src/app/services/git/git.service';
+import { Model, ModelService } from 'src/app/services/model/model.service';
+import { Project } from 'src/app/services/project/project.service';
+import { SessionService } from 'src/app/services/session/session.service';
+import { ModelOptions } from 'src/app/sessions/new-readonly-session-dialog/new-readonly-model-options.component';
+
+@Component({
+  selector: 'new-readonly-session-dialog',
+  templateUrl: './new-readonly-session-dialog.component.html',
+  styleUrls: ['./new-readonly-session-dialog.component.css'],
+})
+export class NewReadonlySessionDialogComponent implements OnInit {
+  constructor(
+    public sessionService: SessionService,
+    private gitService: GitService,
+    private router: Router,
+    public modelService: ModelService,
+    @Inject(MAT_DIALOG_DATA) public data: { project: Project; model: Model }
+  ) {}
+
+  private model_options: ModelOptions[] = [];
+
+  get modelOptions(): ModelOptions[] {
+    return this.model_options;
```

Comment:
```suggestion
    return this.modelOptions;
```

Reply:
Not for the getter...

Reply:
You are right. 


```
  private _modelOptions: ModelOptions[] = [];

  get modelOptions(): ModelOptions[] {
    return this._modelOptions;
```

This would be the better solution then :)